server/channels/app/platform/log.go

```diff
@@ -32,7 +31,10 @@ func (ps *PlatformService) initLogging() error {
 	// create the app logger if needed
 	if ps.logger == nil {
 		var err error
-		ps.logger, err = mlog.NewLogger(logr.MaxFieldLen(*ps.Config().LogSettings.MaxFieldSize))
+		ps.logger, err = mlog.NewLogger(
+			mlog.MaxFieldLen(*ps.Config().LogSettings.MaxFieldSize),
+			mlog.StackFilter("log"),
```

Comment:
Pigg backing this single change in here: It's useful to filter out the std library `log` package as it's more useful to see who actually triggered the log message.